recipes/example/meta.yaml

```diff
@@ -45,6 +45,8 @@ about:
   # Prefer the short version, e.g: GPL-2.0 instead of GNU General Public License version 2.0
   # See https://opensource.org/licenses/alphabetical
   license: MIT
+  # If the license has different variants uncomment and edit the next line.
+  # license_family: BSD, Apache, GPL
```

Comment:
I was not sure if the family be specified in any case. 

I removed the comment and included your suggestion. In theory `license_family` should be self-explaining anyways.